src/ecs/resources/input/input_handler.rs

```diff
@@ -81,6 +28,8 @@ impl<'a> Iterator for Buttons<'a> {
 /// ```
 #[derive(Default)]
 pub struct InputHandler {
+    /// Maps inputs to actions and axes.
+    pub bindings: Bindings,
     pressed_keys: SmallVec<[VirtualKeyCode; 16]>,
```

Comment:
Can it be stored also in one array with state (pressed, released, down) for each `VirtualKeyCode` for lookup purpose?